Makefile

```diff
@@ -21,6 +22,17 @@ windows: clean requirements
 	source .virtualenv/bin/activate
 	python -m nsist windows-installer.cfg
 
+requirementsgui:
+	python3 -m venv .virtualenv
+	source .virtualenv/bin/activate
+	pip3 install --upgrade wheel
+	pip3 install -r requirements.txt
+	pip3 install -r requirementsgui.txt
```

Comment:
I made a separate requirements file for the gui packages so that they don't need to be installed if only the cli is needed.